Inferences/Induction.cpp

```diff
@@ -1363,9 +1390,15 @@ void InductionClauseIterator::performStructInductionFreeVar(const InductionConte
   if (context._indTerms.size() > 1) return;
   TermList sort = SortHelper::getResultSort(context._indTerms[0]);
   TermAlgebra* ta = env.signature->getTermAlgebraOfSort(sort);
+
+  // Synthesis specific:
+  SynthesisALManager* synthMan = static_cast<SynthesisALManager*>((env.options->questionAnswering() == Options::QuestionAnsweringMode::SYNTHESIS) ? SynthesisALManager::getInstance() : nullptr);
+  List<Term*>* fnHeads = List<Term*>::empty();
```

Comment:
Does this *really* have to be a `List`?

Reply:
What other structure do you propose? (I don't have a preference, it just needs to preserve the order.)

Reply:
Fair point. Turns out this also simplified processing down the line. Thanks for the suggestion!